pele_platform/Frag/parameters/files.py

```diff
@@ -83,6 +83,14 @@ def frag_from_core_parameters(parameters, args):
 
         parameters.core_format = parameters.core.rsplit(".")[-1]
 
+        from pathlib import Path
+        import shutil
+
+        new_file = os.path.join(os.path.abspath("."), Path(parameters.core_process).stem + "_frag.pdb")
```

Comment:
```suggestion
        new_file = os.path.join(os.getcwd(), Path(parameters.core_process).stem + "_frag.pdb")
```
A neat way to get the working directory.
Also, we're never getting rid of those pluses, are we? 🤣